Pipfile

```diff
@@ -30,6 +30,7 @@ fakeredis = "*"
 mypy = "*"
 black = "==19.3b0"
 pytest-flask = "*"
+pytest-mock = "*"
```

Comment:
Has this module been introduced as part of tests that have now been removed? 

Looks like its for wrapping mock.patch, which isn't used in any of these tests. I can switch to a standard mock and the tests still pass.

Reply:
Did you uninstall it and run the tests?

Reply:
Yes.

Reply:
Looks like this is a result of using pipenv.

To see the actual dependency on this try:
```
$ pipenv --rm
$ pipenv install --dev
$ make test-unit
```

Reply:
Removing the package from the pipfile and updating doesn't remove it from site-packages, so it's still available to pytest

Reply:
This still works for me when I've removed the module from the pipfile, lock, completely removed/reinstalled the env and replaced mocker with mock.

What behaviour do the new unit tests use from this module?

Reply:
Using the mocker fixture

Reply:
Using the steps you've outlined results in a failure for me locally

Reply:
You don't need to use mock at all, these can be empty List/AnswerStore/dicts?